backend/database.py

```diff
@@ -318,14 +318,21 @@ def get_published_result(session, league_name):
                             table[value_name] = {}
                         table[value_name][result.team.name] = value
 
+            feedback = None
+            if sim.feedback_str is not None:
+                feedback = sim.feedback_str
+            elif sim.feedback_json is not None:
+                feedback = json.loads(sim.feedback_json)
```

Comment:
Usually, when I'm using `if` / `elif` I prefer to use an `else` at the end (python doesn't have the case / switch, but it has the elif, which has the same behaviour, and usually, case / switch has a default value). My suggestion here:

```
            if sim.feedback_str is not None:
                feedback = sim.feedback_str
            elif sim.feedback_json is not None:
                feedback = json.loads(sim.feedback_json)
            else:
                feedback = None
```
It makes more clear that... if the previous conditions arent satisfied, the default value will be None.

Reply:
Makes sense, imlemented